guides.md

```diff
@@ -237,6 +239,140 @@ members. Skip the steps below at your discretion for low-stakes topics.
   1. Use the private dial-in number from our [technical Jitsi documentation](https://github.com/hyphacoop/organizing-private/blob/master/documents/infrastructure/jitsi.md).
   2. _After_ prompt, enter conference code `307 314 3734 #`
 
+## Onboarding Checklist
+
+This is a checklist for onboarding a new Member to the Co-operative 🚀
+
+### Setting up communications
+
```

Comment:
```suggestion

Please populate [employee record](https://link.hypha.coop/employees) during onboarding.

```

Reply:
Worth mentioning it here, and not just below?

Reply:
Or just

```suggestion

See: [employee record](https://link.hypha.coop/employees)

```

Reply:
Why mention twice this specific one?

Reply:
Because each guide is standalone, and someone running through either will be collecting that info or pulling from it. It's a coupled resource. 🤷‍♂